fissa/tests/test_datahandler.py

```diff
@@ -7,9 +7,9 @@
 import tifffile
 
 from .base_test import BaseTestCase
-from .. import datahandler
+from ..datahandler import DataHandler
 from .. import roitools
-
+datahandler = DataHandler()
```

Comment:
I don't like this. `datahandler` is the name of the module and `DataHandler` the name of the class. Should not make the name of the instance of `DataHandler` clash with the name of the module. Need to use a different name for the instance, such as
```suggestion
dh = DataHandler()
```
(which obviously has implications for later code in the file which needs datahandler changed to dh)

Maybe we should consider renaming the module instead though.

Reply:
> I've not read the notebook changes yet because they don't render on GitHub. Presumably the notebook changes are only to do with using a custom datahandler?
Yes. With an explanation of how to use it.

Reply:
> Maybe we should consider renaming the module instead though.
The case of ```datahandler = DataHandler()``` is actually consistent with how use core.Experiment() class (we call exp=fissa.Experiment() or experiment=fissa.Experiment() a few times in the examples). But I agree it is not the best naming. Now I import DataHandler from datahandler, and then make an DataHandler object called datahandler. Yuk.

We could rename the module to ```extraction``` (following Suite2p) and keep the class-name, as well as use your ```dh``` suggestion, so it becomes
```suggestion
from ..extraction import DataHandler()
dh = DataHandler()
```